web_src/css/admin.css

```diff
@@ -42,3 +42,15 @@
 .admin .table th {
   white-space: nowrap;
 }
+
+.admin-responsive-columns {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  margin-bottom: 1rem;
+}
+
+.admin-responsive-columns > .column {
+  flex-grow: 1;
+  flex-basis: calc(50% - .5rem);
```

Comment:
`flex: 1` should all that's necessary for equally sized boxes.

Reply:
Seems that you're right...... changed to `flex: 1`. I started with flex-basis 50%, then added a gap, then tried to make it responsible and in the end I had this :laughing: